rhel/vex/parser.go

```diff
@@ -0,0 +1,394 @@
+package vex
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/package-url/packageurl-go"
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/pkg/cpe"
+	"github.com/quay/claircore/toolkit/types/csaf"
+	"github.com/quay/claircore/toolkit/types/cvss"
+)
+
+// Parse implements [driver.Updater].
+func (u *VEXUpdater) Parse(ctx context.Context, contents io.ReadCloser) ([]*claircore.Vulnerability, error) {
+	// NOOP
+	return nil, nil
```

Comment:
```suggestion
	return nil, errors.ErrUnsupported
```

Returning `(nil, nil)` is a valid result, is that a desired signal?

Reply:
I think better to return an error, if someone calls this they are probably expecting it to do something.